internal/app/wwctl/container/copy/main.go

```diff
@@ -35,7 +36,7 @@ func CobraRunE(cmd *cobra.Command, args []string) (err error) {
 		return
 	}
 
-	err = container.Duplicate(cdp.ContainerSource, cdp.ContainerDestination)
+	err = container.Duplicate(cdp.ContainerSource, cdp.ContainerDestination, cdp.Build)
```

Comment:
@JasonYangShadow rather than nesting build within the `Duplicate` function, could you break it out and have `wwctl container copy` run `Duplicate()` and then, if that's successful and `--build` is set`, then call `Build()` separately? I think that's a cleaner interface.

Reply:
updated, thanks!